### DISCUSSION TITLE
How to detach/disconnect control unit from desk?

### DISCUSSION THREAD

**Discussion by raleighlittles** [gcormier/megadesk#146] [Q&A]
I saw this video on Greg's instructions of how to disassemble the actual control unit: https://www.youtube.com/watch?v=jCPlM2KYwDQ

However, in the video, he _starts off_ with the control unit already detached from the desk.. how do I get to this point? My control unit is still attached to the desk and doesn't have any screws to undo.

![IMG_0887](https://github.com/gcormier/megadesk/assets/2137201/160f4c2c-978e-4eb9-b8e0-d3a20f0c8f1d)

![IMG_0888](https://github.com/gcormier/megadesk/assets/2137201/5ac230f8-6ae1-4cb0-a321-5795a14e2914)

**Comment by gcormier** [comment]
How did you attach the control unit when you built it? :)

**Reply by raleighlittles** [reply]
@gcormier I bought mine assembled unfortunately. In the assembly instructions I found online it looks like there's supposed to be screws attached but mine doesn't have any.

<img width="971" alt="Screenshot 2023-12-27 at 9 28 53 PM" src="https://github.com/gcormier/megadesk/assets/2137201/fc4059a5-06bf-415f-8b6d-9ff36f0858dd">

**Reply by gcormier** [reply]
Yup. You could try to pry it off, someone used adhesive I'd wager. Hopefully your desk top doesn't get damaged.
